openmmtools/alchemy.py

```diff
@@ -1482,37 +1593,36 @@ def _alchemically_modify_PeriodicTorsionForce(reference_force, alchemical_region
             in alchemical_region.
 
         """
-        # Don't create a force if there are no alchemical torsions.
-        if len(alchemical_region.alchemical_torsions) == 0:
-            return {'': [copy.deepcopy(reference_force)]}
+        if cumulative_torsion_force == None:
+            force = copy.deepcopy(reference_force)
+        else:
+            force = copy.deepcopy(cumulative_torsion_force)
 
-        # Create PeriodicTorsionForce to handle unmodified torsions.
-        force = openmm.PeriodicTorsionForce()
-        force.setForceGroup(reference_force.getForceGroup())
+        if len(alchemical_region.alchemical_torsions) == 0:
+            return {'': [copy.deepcopy(force)]}
 
         # Create CustomTorsionForce to handle alchemically modified torsions.
-        energy_function = "lambda_torsions*k*(1+cos(periodicity*theta-phase))"
+        energy_function = "lambda_torsions_{}*k*(1+cos(periodicity*theta-phase))".format(alchemical_region.name)
```

Comment:
I think this will result in the variable being called `lambda_torsions_None` if `alchemical_region.region_name` is not set. It should be `lambda_torsions` (without the trailing `_` character) in this case.

The same comment applies for all the lambda variable names in the energy expressions below.